### PR TITLE
Add MIR referrer details for each SDK

### PR DESCRIPTION
**Pull request by Sporiff**
Related to https://adjustcom.atlassian.net/browse/ADAPP-17470

### CODE REVIEW

**Comment by Sporiff** [comment]
@shashanksu @uerceg This update relates to the conversation on https://adjustcom.atlassian.net/browse/ADAPP-17470. I've added MIR instructions for each of the platforms that we have documented in this repo in line with what you've noted on that ticket. Could you take a look and let me know if anything looks amiss?

- Android: https://dev-docs-git-mir-referrer-update-product-content.vercel.app/en/sdk/android#meta-referrer-integration
- Unity: https://dev-docs-git-mir-referrer-update-product-content.vercel.app/en/sdk/unity#meta-referrer-integration
- React-Native: https://dev-docs-git-mir-referrer-update-product-content.vercel.app/en/sdk/react-native#meta-referrer-integration
- Flutter: https://dev-docs-git-mir-referrer-update-product-content.vercel.app/en/sdk/flutter#meta-referrer-integration

File: src/content/docs/sdk/android/index.mdx [hunk 1]
```diff
@@ -143,13 +143,50 @@ The Xiaomi referrer plugin enables you to record install referrer values for app
...
+
+1. Find your Meta app ID in your [App Dashboard](https://developers.facebook.com/apps). See Meta's [App Dashboard documentation](https://developers.facebook.com/docs/development/create-an-app/app-dashboard) for more information.
+2. Pass your App ID as a __string__ argument to the `AdjustConfig.setFbAppId` method.
+
+<Tabs>
+<Tab title="Kotlin" sync="kotlin">
+
+```kotlin {2}
+val config = AdjustConfig(this, appToken, environment)
+config.setFbAppId(<FB_APP_ID_STRING>);
```
Comment (shashanksu) [inline src/content/docs/sdk/android/index.mdx:168]: semicolon `;` is not required.

File: src/content/docs/sdk/android/index.mdx [hunk 2]
```diff
@@ -143,13 +143,50 @@ The Xiaomi referrer plugin enables you to record install referrer values for app
...
+Adjust.onCreate(config)
+```
+
+</Tab>
+<Tab title="Java" sync="java">
+
+```java {2}
+AdjustConfig config = new AdjustConfig(this, appToken, environment);
+config.setFbAppId(<FB_APP_ID_STRING>)
```
Comment (shashanksu) [inline src/content/docs/sdk/android/index.mdx:177]: semicolon `;` is required at the end.
```
config.setFbAppId(<FB_APP_ID_STRING>);
```

**Comment by shashanksu** [comment]
@Sporiff Changes looks good to me.  But let @uerceg also review it especially for non-natives SDK and confirm.

**Review by uerceg — APPROVED** [review]

**Pull request merged**